week-3/quote-generator/quotes.js

```diff
@@ -25,469 +25,509 @@ function pickFromArray(choices) {
   },
   {
     quote:
       "Our lives begin to end the day we become silent about things that matter.",
-    author: "Martin Luther King Jr.",
+    author: "- Martin Luther King Jr.",
   },
   {
     quote: "Do what you can, where you are, with what you have.",
-    author: "Teddy Roosevelt",
+    author: "- Teddy Roosevelt",
   },
   {
     quote:
       "If you do what you've always done, you'll get what you've always gotten.",
-    author: "Tony Robbins",
+    author: "- Tony Robbins",
   },
   {
     quote: "Dreaming, after all, is a form of planning.",
-    author: "Gloria Steinem",
+    author: "- Gloria Steinem",
   },
   {
     quote:
       "It's your place in the world; it's your life. Go on and do all you can with it, and make it the life you want to live.",
-    author: "Mae Jemison",
+    author: "- Mae Jemison",
   },
   {
     quote:
       "You may be disappointed if you fail, but you are doomed if you don't try.",
-    author: "Beverly Sills",
+    author: "- Beverly Sills",
   },
   {
     quote: "Remember no one can make you feel inferior without your consent.",
-    author: "Eleanor Roosevelt",
+    author: "- Eleanor Roosevelt",
   },
   {
     quote: "Life is what we make it, always has been, always will be.",
-    author: "Grandma Moses",
+    author: "- Grandma Moses",
   },
   {
     quote:
       "The question isn't who is going to let me; it's who is going to stop me.",
-    author: "Ayn Rand",
+    author: "- Ayn Rand",
   },
   {
     quote:
       "When everything seems to be going against you, remember that the airplane takes off against the wind, not with it.",
-    author: "Henry Ford",
+    author: "- Henry Ford",
   },
   {
     quote:
       "It's not the years in your life that count. It's the life in your years.",
-    author: "Abraham Lincoln",
+    author: "- Abraham Lincoln",
   },
   {
     quote: "Change your thoughts and you change your world.",
-    author: "Norman Vincent Peale",
+    author: "- Norman Vincent Peale",
   },
   {
     quote:
       "Either write something worth reading or do something worth writing.",
-    author: "Benjamin Franklin",
+    author: "- Benjamin Franklin",
   },
   {
     quote: "Nothing is impossible, the word itself says, “I'm possible!”",
-    author: "-Audrey Hepburn",
+    author: "- Audrey Hepburn",
   },
   {
     quote: "The only way to do great work is to love what you do.",
-    author: "Steve Jobs",
+    author: "- Steve Jobs",
   },
   {
     quote: "If you can dream it, you can achieve it.",
-    author: "Zig Ziglar",
+    author: "- Zig Ziglar",
   },
 ];
 
-// call pickFromArray with the quotes array to check you get a random quote
+
+
+function getQoutes(evet){
+const quoteParagahraph = document.getElementById("quote");
+const currentQuote = pickFromArray(quotes);
+quoteParagahraph.innerText = currentQuote.quote;
+const quoteParagahraph2 = document.getElementById("author");
+quoteParagahraph2.innerText = currentQuote.author;
```

Comment:
Great job! In the future, consider using more meaningful variable names like `authorName` for better clarity and readability.